src/pages/create/recruitCreatePage/RecruitCreatePage.tsx

```diff
@@ -1,32 +1,30 @@
-import React, { useEffect } from 'react';
+import React from 'react';
 import S from './RecruitCreatePage.styled';
-import { TitleAndIntro } from '../../../components/index';
-import RecruitInfoWrapper from './wrappers/RecruitInfoWrapper';
-import RecruitPostWrapper from './wrappers/RecruitPostWrapper';
+import {
+	BasicInformation,
+	ControlButtons,
+	Description,
+	DetailedInformation,
+	RecruitRoles,
+} from '../../../components/index';
 import { useMutation } from '@tanstack/react-query';
-import { useRecoilValue, useRecoilState } from 'recoil';
+import { useRecoilState, useSetRecoilState, useRecoilValue } from 'recoil';
 import { recruitInputState, validState } from '../../../atom';
-import { postingRecruit } from '../../../api';
+import { postingRecruit } from '../../../service/recruit/posting';
 import { useNavigate } from 'react-router-dom';
-
-const descriptions = [
-	'함께할 멤버들에게 알릴 기본 정보들을 기입해주세요!',
-	'기본 정보를 기반으로 구인글을 제공할 수 있습니다.',
-];
-
-const introductions = [
-	'게시될 구인글을 작성해주세요!',
-	'어떤 방식으로 진행하고 싶은지 구체적으로 작성해주세요.',
-];
+import RecruitTags from '../../../components/recruit/create/RecruitTags';
+import { InputState } from '../../../types';
 
 const RecruitCreatePage = () => {
 	const navigate = useNavigate();
-	const recruitFormData = useRecoilValue(recruitInputState);
-	const [isSubmit, setIsSubmit] = useRecoilState(validState);
-	const onClickCancel = () => {
-		// 모달창 띄워서 한 번 더 확인시키고 이동하기
-		// navigate('/');
-	};
+	const setIsSubmit = useSetRecoilState(validState);
+	const [formData, setFormData] = useRecoilState<InputState>(recruitInputState);
+	const validCheck = useRecoilValue(validState);
+
```

Comment:
```suggestion
const postAvailable =
		validCheck.isCategory &&
		validCheck.isDeadline &&
		validCheck.isProcedure &&
		validCheck.isScope &&
		validCheck.isTag &&
		validCheck.isTitle;
```

해당 상수를 함수 바깥으로 빼어도 정상작동하길래 분리하는게 좋아 보입니다!

Reply:
이 부분은 일부로 해당 로직 안에 넣었습니다!!
RecruitCreatePage가 재렌더링될 때마다 postAvailable을 계산하기 때문에
submit을 할 때만 연산할 수 있도록 해당 함수에 넣어놓았습니다!!

상수는 사실 컴포넌트 밖에서 선언하는게 가장 좋지만, validCheck이 컴포넌트 속에 있기 때문에 해당 함수에서 선언하는게 현재로써는 최선이라고 생각했습니다!

Reply:
아하!! 그렇군요!! 이해했습니당!!👍🏻